### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by mdennis1318** [mdennis1318/MI-449-SS17-740-js-serving-content-with-express-OctGcX#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://dennis57-kfxl35erbteu5octgcx.herokuapp.com/) and meets the following criteria:

- [x] No code is repeated in your files; create a partial for any shared elements (`nav`, `head`, `header`, etc.)
- [x] Make a navigation to move throughout your site, using `forEach` to create a set of links to other data pages
- [x] Each item should have an object with an image, a link, a title, and more properties based on your topic (i.e. sandwich ingredients, movie release date, etc.)
- [x] Include at least 3 data pages and a homepage; routes should be created in Express for each
- [x] Use data appropriately with HTML tags (paragraphs in `p` tags, headings in `h1`-`h6`, etc.)
- [x] Use `express.static` to create a folder for static assets (images, stylesheets, etc.)
- [x] Include a `.gitignore` file so that `node_modules` and any other unnecessary files are not commited to GitHub

**Comment by stuartpearman** [comment]
Nice, I like what you have going Matt, great job!

Since your 4 soccer pages all have the same basic structure, see if you can make them with just one `.ejs` file. You would still need a separate file for your homepage layout.

You could also combine the navs and the teams object into one, and use that to render both the page content and the navigation.

Let me know if you have any questions!

**Comment by mdennis1318** [comment]
@stuartpearman Hi Stuart,

I made all of the content pages into a partial and used that to render all the teams info. I kept the navigation and team info separate because i have to render the navigation for the homepage too. If there is another way I should go about that let me know!

Thanks!

**Comment by stuartpearman** [comment]
Hi Matt,

You're so close, I'm happy with what you've done here but I think if we can make it just a bit better it'll be a bit of a breakthrough.

First thing, instead of making separate objects for your trophies and your teams, you could combine them like so:

```
var realmadrid = {
  name: 'Real Madrid',
  website: 'http://www.realmadrid.com/en',
  logo: '/images/Real_Madrid.png',
  league: 'La Liga',
  stadium: 'Santiago Bernabéu Stadium',
  coach: 'Zinedine Zidane',
  trophies: {
    name: 'Real Madrid',
    website: 'http://www.realmadrid.com/en',
    logo: '/images/Real_Madrid.png',
    league: 'La Liga',
    stadium: 'Santiago Bernabéu Stadium',
    coach: 'Zinedine Zidane'
  }
}
```

Second, see what happens if you do this:

```
app.get('/barcelona', function (request, response) {
  response.render('pages/chelsea', {
    navs: navs,
    teaminfo: barcelona,
    trophies: barcaTrophies
  })
})
```

I swapped this:

```
response.render('pages/barcelona', { ... })
```

with

```
response.render('pages/chelsea', { ... })
```

I bet it will work if you try. Once you get that working, try rendering _all_ of your teams with just one `ejs` file.

You're doing great things! Reach out with questions if you have them.

**Comment by mdennis1318** [comment]
@stuartpearman Hey Stuart,

I fixed up server.js and put trophies in the team info variables. Also I got all the pages to render off of one file! That is so smart, wish I would have thought of it. I deleted the teaminfo.ejs partial because I thought it would be better to just use the one file for the team pages. Please take another look and let me know if anything else could be better!

Thanks!

**Comment by stuartpearman** [comment]
Nope, it looks great! I'm glad you enjoyed it, you've definitely earned yourself a ship it squirrel :tada: :shipit:
